src/Airbrake/Notice.class.php

```diff
@@ -264,7 +264,7 @@ private static function argToString($arg, Configuration $configuration, $level =
         }
 
         if (strlen($result) > $maxLength) {
-            $result = substr($result, 0, self::MAX_SINGLE_ARG_STRING_LENGTH);
+            $result = mb_strcut($result, 0, self::MAX_SINGLE_ARG_STRING_LENGTH);
```

Comment:
Looks like the max length is kind of arbitrary, so you could have kept `mb_substr` instead of replacing by `mb_strcut`… I prefer `cut` indeed, but it really doesn't matter anyway, and an argument here is pointless for ±1 char in some cases.